src/pages/events/configure-commerce.md

```diff
@@ -143,7 +143,7 @@ You must configure Commerce to communicate with your project. You will need two
 
 1. Copy the contents of the `<workspace-name>.json` file into the **Adobe I/O Workspace Configuration** field.
 
-1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value can be any unique string.
+1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value must contain English alphanumeric characters, underscores (_), and hyphens (-) only.
```

Comment:
```suggestion
1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value can only contain English alphanumeric characters, underscores (_), and hyphens (-).
```
minor suggestion, it is probably fine as is, but i don't like "only" being that far away for some reason